packages/ibc-utils/src/neutron.rs

```diff
@@ -0,0 +1,168 @@
+use std::collections::BTreeMap;
+
+use cosmos_sdk_proto::cosmos::base::v1beta1::Coin;
+use cosmos_sdk_proto::traits::MessageExt;
+use cosmwasm_std::{to_json_string, Binary, CosmosMsg, DepsMut, Env, StdError, StdResult, Uint128};
+use neutron_sdk::{
+    bindings::{msg::IbcFee, query::NeutronQuery},
+    query::min_ibc_fee::query_min_ibc_fee,
+};
+
+use crate::types::{ForwardMetadata, PacketForwardMiddlewareConfig, PacketMetadata};
+
+// Default timeout for IbcTransfer is 300 blocks
+const DEFAULT_TIMEOUT_HEIGHT: u64 = 300;
+// Default timeout for IbcTransfer is 600 seconds
+const DEFAULT_TIMEOUT_TIMESTAMP: u64 = 600;
+const NTRN_DENOM: &str = "untrn";
+
+#[allow(clippy::too_many_arguments)]
+pub fn ibc_send_message(
+    deps: DepsMut<NeutronQuery>,
+    env: Env,
+    channel: String,
+    port: Option<String>,
+    sender: String,
+    to: String,
+    denom: String,
+    amount: u128,
+    memo: String,
+    timeout_height: Option<u64>,
+    timeout_timestamp: Option<u64>,
+    denom_to_pfm_map: BTreeMap<String, PacketForwardMiddlewareConfig>,
+) -> StdResult<CosmosMsg> {
+    // contract must pay for relaying of acknowledgements
+    // See more info here: https://docs.neutron.org/neutron/feerefunder/overview
```

Comment:
nit: contract is not paying anymore right, it's the base account now.

Reply:
Yeah .. but the account is a contract, right ? 😁